packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -140,7 +140,8 @@ export const dropdownStyles: ComponentSlotStylesPrepared<DropdownStylesProps, Dr
 
   selectedItems: ({ props: p, variables: v }): ICSSInJSStyle => ({
     display: 'flex',
-    flexWrap: 'wrap',
+    minHeight: pxToRem(28),
+    marginBottom: pxToRem(4),
```

Comment:
Why do you need these values ? setting the width and height of the trigger button should already extend the button down the right hand side of the dropdown

Reply:
I understand the bottom margin since it will cut off the some of the chips, but why the min-height ?

Reply:
To keep the current height otherwise it will be smaller than before

Reply:
Makes sense, thanks